packages/frontend/src/store/actions.js

```diff
@@ -160,10 +169,12 @@ export default {
         const user = res.data
 
         if (!user) {
+          dispatch('failedUserLogin')
```

Comment:
When it's this simple, you can just call the commit directly. No need for an action at all. You can delete that.
```suggestion
          commit('loginFail', true)
```

Reply:
Changes made